db/mysql-instance.yml

```diff
@@ -171,17 +172,25 @@ Resources:
         - Ref: AWS::NoValue
       Engine: MySQL
       EngineVersion: 5.5.46
-      MasterUsername: !Ref DBUser
-      MasterUserPassword: !Ref DBPassword
+      MasterUsername:
+        Fn::If:
```

Comment:
Let's switch these to the short form `If` to be consistent with our other templates and then I think this should be good for now (I think there's five total)